volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -234,6 +234,10 @@ def get_end(self):
 
         raise AttributeError("Unable to find the ending VPN member")
 
+    def get_size(self) -> int:
```

Comment:
Thanks, this is an additive change, which I think therefore requires the minor version number to be bumped to 2.4.0 please (and the four plugins that make use of it to bump their `_required_framework_version`).  Otherwise it looks good, and happy to merge after that...  5:)

Reply:
I'd quite like to get this merged before the impending code freeze.  It should just be a case of bumping version numbers and then it's good to merge...  5:)

Reply:
Hello @ikelos, if he's busy I'm willing to take care of the needs of this branch.
However, in this situation, there seems to be no way to work on the branch where PR is open in this Repository.
Is it possible to work on PR only by the person who suggests it?

Reply:
Hi @digitalisx, I'm afraid so.  You'd need to construct your own pull request which would then be worked on separately (and obsolete the original PR).  Luckily it looks like this has seen some progress and should be good to commit...  5:)